packages/remix-express/server.ts

```diff
@@ -89,20 +89,21 @@ export function createRemixHeaders(
   return headers;
 }
 
-export function createRemixRequest(req: express.Request): NodeRequest {
+export function createRemixRequest(
+  req: express.Request,
+  res: express.Response
+): NodeRequest {
   let origin = `${req.protocol}://${req.get("host")}`;
   let url = new URL(req.url, origin);
 
+  // Abort action/loaders once we can no longer write a response
   let controller = new AbortController();
-
-  req.on("close", () => {
-    controller.abort();
-  });
+  res.on("close", () => controller.abort());
 
   let init: NodeRequestInit = {
     method: req.method,
     headers: createRemixHeaders(req.headers),
-    signal: controller.signal,
+    signal: controller.signal as AbortSignal,
```

Comment:
It seems our the `abort-controller` polyfill is missing `reason` and `throwIfAborted` props :(

https://github.com/mysticatea/abort-controller/issues/33
https://github.com/mysticatea/abort-controller/issues/36

I wonder if there's an alternative we could look for?  We don't reference either of those fields at the moment .... but this feels like a ticking time bomb since our types tell us we _could_